packages/assurance/README.md

```diff
@@ -82,7 +86,7 @@ public class MainApplication extends Application implements ReactApplication {
 }
 ```
 
-To connect to a griffon session by scanning the QR code. Follow the [Android documentation](https://developer.android.com/training/app-links/deep-linking) on information about how to setup a deeplink.
+To connect to an Assurance session by scanning the QR code. Follow the [Android documentation](https://developer.android.com/training/app-links/deep-linking) on information about how to setup a deeplink.
```

Comment:
"To connect to an Assurance session by scanning the QR code, follow the [Android documentation](https://developer.android.com/training/app-links/deep-linking) for more information about how to setup a deeplink."